plugins/modules/source_control/github/github_repo.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Álvaro Torres Cogollo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from github import Github, GithubException
```

Comment:
You need to use import guards. See for example the gitlab_hook module:
- https://github.com/ansible-collections/community.general/blob/main/plugins/modules/source_control/gitlab/gitlab_hook.py#L167-L173
- https://github.com/ansible-collections/community.general/blob/main/plugins/modules/source_control/gitlab/gitlab_hook.py#L346-L347